doc/source/io.rst

```diff
@@ -282,16 +285,16 @@ float_precision : string, default None
   Specifies which converter the C engine should use for floating-point values.
   The options are ``None`` for the ordinary converter, ``high`` for the
   high-precision converter, and ``round_trip`` for the round-trip converter.
-lineterminator : str (length 1), default ``None``
```

Comment:
I would add a note (``.. note:: ``) somewhere on this page (maybe add the end of the enumeration of all the options), explaining that for some keywords aliases without underscores are accepted for historical reasons, but that it is recommended to use the version with underscores.